src/routes/RequestsRoute.js

```diff
@@ -1037,10 +1072,11 @@ class RequestsRoute extends React.Component {
     this.setState({ errorModalData: {} });
   };
 
-  getPrintTemplate() {
+  getPrintTemplate(slipType) {
     const staffSlips = get(this.props.resources, 'staffSlips.records', []);
-    const pickSlip = staffSlips.find(slip => slip.name.toLowerCase() === pickSlipType);
-    return get(pickSlip, 'template', '');
+    const slipTemplate = staffSlips.find(slip => slip.name.toLowerCase() === slipType.toLowerCase());
```

Comment:
move `slipType.toLowerCase()` to variable please, to not trigger `toLowerCase()` on each slip element finding

Reply:
Fixed